Rocket.Chat/Controllers/Subscriptions/SubscriptionsList/SubscriptionsViewController.swift

```diff
@@ -573,33 +323,19 @@ extension SubscriptionsViewController: UITableViewDataSource {
 extension SubscriptionsViewController: UITableViewDelegate {
```

Comment:
I believe we can remove this extension all-together.

Reply:
Why?

Reply:
Sorry marked the incorrect extension. It's declaration in not visible on the diff. 😅 
I meant the one starting at line 359.